data/registry_model/test/test_interface.py

```
@@ -1140,3 +1140,286 @@ def test_tag_names_for_manifest(initialized_db, registry_model):
                 found_tag = registry_model.get_repo_tag(repo_ref, found_name)
                 assert registry_model.get_manifest_for_tag(found_tag) == manifest
     assert verified_tag
+
+
+class TestSignatureTagLayerMerging:
+    """Tests for the signature tag layer merging functionality."""
+
+    def _create_oci_manifest_with_layers(self, oci_model, repository_ref, layer_digests):
```

Ruff (0.14.13)

Unused method argument: `oci_model`

(ARG002)